packages/fuels-types/src/lib.rs

```diff
@@ -8,7 +8,7 @@ pub mod block;
 pub mod chain_info;
 pub mod coin;
 pub mod constants;
-mod core;
+pub mod core;
```

Comment:
Everything from core is publicly exported here to provide a uniform type path access in order to simplify abigen.

There is no need for this mod to be made public.